packages/ui/components/ui/dutch-auction-component/progress-bar/indicator/get-progress.test.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, it } from 'vitest';
+import { getProgress } from './get-progress';
+
+describe('getProgress()', () => {
```

Comment:
Moved from one folder up. The only new tests here are the ones involving `seqNum`, but git thinks it's a whole new file.